gui/src/context/taipyReducers.ts

```diff
@@ -264,8 +267,16 @@ export const initializeWebSocket = (socket: Socket | undefined, dispatch: Dispat
             sendWsMessage(socket, "ID", "TaipyClientId", id, id);
             dispatch({ type: Types.SocketConnected });
         });
+        // try to reconnect on connect_error
+        socket.on("connect_error", () => {
+            setTimeout(() => {
+                socket.connect();
```

Comment:
I suppose this will generate another 'connect' event ?
Should we then test the isSocketConnected in the connect event listener ?